appveyor.yml

```diff
@@ -1,16 +1,16 @@
 environment:
   matrix:
-  - julia_version: 1.0
-  - julia_version: 1
-  - julia_version: nightly
+    - julia_version: 1.3
+    - julia_version: 1.4
+    - julia_version: nightly
 
 platform:
   - x64
   - x86
 
 matrix:
   allow_failures:
-  - julia_version: nightly
+    - julia_version: nightly
 
```

Comment:
This (and above) is not how yaml is apparently indented according to https://yaml.org/. Not sure if it has effect though.